docs/integrations/vercel-content-link.mdx

```diff
@@ -94,7 +125,9 @@ const { cleaned, encoded } = vercelStegaSplit(text)
 
 ### Blocks and array fields
 
-All `blocks` and `array` fields by definition do not have plain text strings to encode. For this reason, they are given an additional `_encodedSourceMap` property, which you can use to enable Content Link on entire _sections_ of your site. You can then specify the editing container by adding the `data-vercel-edit-target` HTML attribute to any top-level element of your block.
+All `blocks` and `array` fields by definition do not have plain text strings to encode. For this reason, they are automatically given an additional `_encodedSourceMap` property, which you can use to enable Content Link on entire _sections_ of your site.
+
+You can then specify the editing container by adding the `data-vercel-edit-target` HTML attribute to any top-level element of your block.
```

Comment:
A code example of how to implement this in both the `app` and `pages` world might help. Specially regarding how to access `_encodedSourceMap` property since the field one would be using is a rich text field with embed blocks. How does one access the blocks inside the RT field?

Reply:
Totally. The process is the same for both routers, however, rich text fields do not currently return the `_encodedSourceMap` property. Will add this as a feature and update the docs accordingly.